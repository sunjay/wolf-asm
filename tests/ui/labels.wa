section .code
  add $1, $2
bar:
  add $1, $2
  add $1, $2
  add $1, $2
  add $1, $2
  add $1, $2
  add $1, $2
bar2:
  add $1, $2
  add $1, $2
  # Existing labels
  add $1, bar
  add $1, bar2
  # Non-existent label
  add $1, does_not_exist
  add $1, $2
