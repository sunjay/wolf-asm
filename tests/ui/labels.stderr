[../tests/ui/labels.wa:17:11-17:24] error: unknown label `does_not_exist`

error: aborting due to 1 previous error

